draft-ietf-quic-transport.md

```diff
@@ -1728,9 +1728,9 @@ disable_migration (0x0009):
 
 max_ack_delay (0x000c):
 
-: An 8 bit unsigned integer value indicating the maximum amount of time in
-  milliseconds by which it will delay sending of acknowledgments.  If this
-  value is absent, a default of 25 milliseconds is assumed.
+: The maximum delay (in milliseconds) used to send acknowledgements, encoded as
+  an 8-bit unsigned integer.  If this value is absent, a default of 25
```

Comment:
Nor do you use a delay to send acknowledgments.  Why not simply s/it/the endpoint/ ?